src/site/content/en/blog/betty-crocker/index.md

```diff
@@ -0,0 +1,340 @@
+alt: Betty Crocker Cake Mix—Coffee Cinnamon.
+description: |
+  Case study of cooking site BettyCrocker.com's experiences with implementing the Wake Lock API.
+tags:
+  - post # post is a required tag for the article to show up in the blog.
+  - case-study
+  - wake-lock
+  - capabilities
+  - fugu
+draft: true
+---
+For nearly a century, Betty Crocker has been America's source for modern cooking instruction
+and trusted recipe development.
+Launched in 1997, their site [BettyCrocker.com](https://www.bettycrocker.com/)
+today receives more than 12&nbsp;million visitors per month.
+After they **implemented the Wake Lock API**, their
+**indicators of purchase intent were about 300% higher**
+for wake lock users compared to all users.
+
+## The retired iOS and Android apps
+
+Released to [much fanfare](https://consumergoods.com/betty-crocker-launches-cookbook-app) in 2014,
+Betty Crocker recently took their native apps out of the Apple App Store and the Google Play Store
+after it had been mostly neglected and deprioritized.
+For a long time, the Betty Crocker team has preferred adding new features to the mobile site
+instead of the native apps.
+The technical platform the native apps were created on was outdated,
+and the business did not have the resources
+to support updating and maintaining the apps moving forward.
+In addition to that, the web app also was objectively way bigger traffic-wise,
+more modern, and easier to enhance.
+
+The native apps did have one *killer feature*, though, that its users loved:
+
+> Millennial cooking pro tip: the [@BettyCrocker](https://twitter.com/BettyCrocker) mobile app
+  doesn't dim or lock when you're following a recipe.
+  […]—[@AvaBeilke](https://twitter.com/AvaBeilke/status/996746473168670720)
+
+> 80% of people cook with a device in the kitchen, but screen dimming and locking is a problem.
+  What did [@BettyCrocker](https://twitter.com/BettyCrocker) do?
+  Updated their app to NOT dim when users are in a recipe.
+  […]—[@Katie_Tweedy_](https://twitter.com/Katie_Tweedy_/status/996746567762763776)
+
+## The Wake Lock API
+
+When cooking with a device, there is nothing more frustrating
+than having to touch the screen with messy hands or even your nose when the screen turns off.
+Betty Crocker asked themselves how they could port the killer feature of their native app
+over to the web app.
+This is when they learned about
+[Project Fugu](https://developers.google.com/web/updates/capabilities) and the
+[Wake Lock API](https://web.dev/wakelock/).
+
+![Doughy fingers](doughy-fingers.jpg)
+
+The Wake Lock API provides a way to prevent the device
+from dimming or locking the screen.
+This capability enables new experiences that, until now, required a native app.
+The Wake Lock API reduces the need for hacky and potentially power-hungry workarounds.
+
+### Requesting a wake lock
+
+To request a wake lock, you need to call the `navigator.wakeLock.request()` method
+that returns a `WakeLockSentinel` object.
```

Comment:
```suggestion
that returns a `WakeLockSentinel` object. You'll use this object as a
[sentinel value](https://en.wikipedia.org/wiki/Sentinel_value).
```
Our audience probably knows what a sentinel is but I don't think it hurts to provide a link for the minority that have never heard that idea before